be/src/Project/Services/DownloadCenter/DownloadCenter.Api/DependencyInjection/ConfigurationRoot.cs

```diff
@@ -0,0 +1,52 @@
+﻿using DownloadCenter.Api.Infrastructure;
+using Foundation.Localization.Definitions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace DownloadCenter.Api.DependencyInjection
+{
+    public static class ConfigurationRoot
+    {
+        public static void ConfigureSettings(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.Configure<LocalizationSettings>(configuration);
+        }
+
+        public static void ConfigureDatabaseMigrations(this IApplicationBuilder app, IConfiguration configuration)
```

Comment:
**suggestion:** IConfiguration configuration isn't used anywhere. Let's get rid of it.